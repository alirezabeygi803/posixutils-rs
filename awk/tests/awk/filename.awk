$1 == 1 {
    print FILENAME
}

END {
    print FILENAME
}