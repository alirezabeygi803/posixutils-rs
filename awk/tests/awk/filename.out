tests/awk/test_data.txt
tests/awk/test_data2.txt
tests/awk/test_data2.txt
